docs/apis/cloud-events/cloud-events-overview.md

```diff
@@ -0,0 +1,17 @@
+---
+id: cloud-events-overview
+title: Cloud Events Overview 
+sidebar_label: Overview 
+description: "This section provides an overview of the different Cloud Event commands which are supported by Miranum Connect."
+---
+# Overview
+
+CloudEvents is a [specification](https://github.com/cloudevents/spec/blob/main/cloudevents/spec.md) for describing event 
+data in a common way. CloudEvents seeks to dramatically simplify event declaration and delivery across services, platforms, and beyond!
+
+CloudEvents is a new effort, and it's still under active development. 
+However, its working group has received a surprising amount of industry interest, ranging from major cloud providers to 
+popular SaaS companies. The specification is now under the [Cloud Native Computing Foundation](https://cncf.io/).
+
+**Miranum Connect** uses this standard to enable other software components to use its functionality / APIs. 
```

Comment:
How's that? How can I send cloud events to miranum? How are they delivered to miranum? Do I need a gateway for that?

Reply:
I have not used Cloud Events yet myself... So that is also something I am unsure about myself. 
Miranum has a Spring Cloud dependency. Maybe this allows functionality to trigger it when a cloud event is being sent. 
Lmk if you have a better picture about this topic. :)

Reply:
FYI: We decided to tackle this later once Miranum features the required gateways